R/vaccine_parameters.R

```diff
@@ -1,35 +1,106 @@
-#' @title Parameterise an RTS,S strategy
+#' @title Parameterise an RTS,S epi strategy
+#'
+#' @description distribute RTS,S vaccine when an individual becomes a certain
+#' age. Efficacy will take effect after the last dose
+#'
+#' @param parameters a list of parameters to modify
+#' @param start timestep to turn on epi vaccination
+#' @param end timestep to turn off epi vaccination
+#' @param coverage the coverage for the starter doses
+#' @param age for the target population, (in timesteps)
+#' @param min_wait the minimum acceptable time since the last vaccination (in
+#' timesteps); This includes seasonal boosters and vaccinations between EPI and mass
+#' strategies
+#' @param boosters the timesteps (following the final dose) at which booster vaccinations are administered
+#' @param booster_coverage the proportion of the vaccinated population who will
+#' receive each booster vaccine
+#' @param seasonal_boosters logical, if TRUE the first booster timestep is
```

Comment:
... otherwise they are relative to rtss,s dose 3

Reply:
I like it as it is. Because the rtss_dose doesn't have to be of length 3